.circleci/config.yml

```diff
@@ -63,10 +63,24 @@ jobs:
       - run:
           working_directory: ~/all-of-us/raw-data-repository
           command: 'sudo service mysql status || sudo service mysql start; '
+      - restore_cache:
```

Comment:
So for restore_cache / save_cache I think we want something like this:

https://github.com/all-of-us/pyclient/blob/master/.circleci/config.yml#L15

https://github.com/all-of-us/pyclient/blob/master/.circleci/config.yml#L28

So that we save off venv and don't have to fetch all the dependencies every time. (If you want to do that in a follow-up, that's fine.) I'm not sure that the current code with restore_cache without a corresponding save_cache is actually doing anything...

Reply:
done